src/messages/ImageMessage.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div class='sc-message--image' :style="messageColors">
+    <div class="sc-message--image-icon">
+      <img :src="data.file.url" class="sc-image">
+    </div>
+    <div class='sc-message--image-name' :style="messageColors">
+      <a :href="data.file.url ? data.file.url : '#'" target='_blank'>{{data.file.name || ''}}</a>
+    </div>
+    <div class="sc-message--file-text" :style="messageColors">{{data.text}}<p v-if="data.meta" class='sc-message--meta' :style="messageColors">{{data.meta}}</p></div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    data: {
+      type: Object,
+      required: true
+    },
+    messageColors: {
+      type: Object,
+      required: true
+    }
+  }
+}
+</script>
+
+<style scoped>
+.sc-message--file {
```

Comment:
These styles are selecting `.sc-message--file` elements, but there are none in this component. Please clean up.